.github/CONTRIBUTING.md

```diff
@@ -1,28 +1,39 @@
 # Contributing guidelines
 
-For anyone who knows JavaScript basics: feel free to contribute to this repository! Our code is open source. Be sure to follow our [code of conduct](https://github.com/ScratchAddons/ScratchAddons/blob/master/.github/CODE_OF_CONDUCT.md).
+Whether you want to send us feedback, fix a problem, create a new addon, or something else, feel free to contribute! Our code is open source.
 
-## Filing [issues](https://docs.github.com/en/github/managing-your-work-on-github/about-issues)
+You can learn more about the types of contributions you can make [here](https://scratchaddons.com/docs/faq/#contributing).
 
-Found a bug? Report it in an issue! But first, be sure to check if it was already reported by using the search bar [here](https://github.com/ScratchAddons/ScratchAddons/issues). If there are no similar issues, you can create a new one. We will take a look at it.
+Be sure to follow our [code of conduct](https://github.com/ScratchAddons/ScratchAddons/blob/master/.github/CODE_OF_CONDUCT.md).
 
-Also, feel free to ask questions, suggest new features, and more.
+## Reporting bugs and suggesting features
 
-Another way to suggest an idea or to ask a question is to start a [discussion](https://github.com/ScratchAddons/ScratchAddons/discussions).
+Found a bug? Have an idea? You can open an [issue](https://github.com/ScratchAddons/ScratchAddons/issues) to report bugs and send feedback! But first, be sure to [search for existing issues](https://github.com/ScratchAddons/ScratchAddons/issues). If there are no similar issues, you can create a new one. We will take a look at it.
 
-## Creating [pull requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+You can also suggest ideas or ask questions in [Discussions](https://github.com/ScratchAddons/ScratchAddons/discussions).
 
-Contributions are welcome! If you know how to fix an issue, you can create a pull request with the necessary changes that we can then add to Scratch Addons.
+## Contributing code
 
-If you're making a new addon, our [FAQ page](https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/) will teach you how to make one.
+If you are familiar with JavaScript, HTML, and CSS, you can contribute by fixing bugs, making adjustments, or adding features. Just follow the steps below to open a pull request to propose your changes which we can then add to Scratch Addons.
+
+### Learn more about making addons
+
+If you're making a new addon, our ["Creating an Addon" guide](https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/) will teach you how to make one.
+
+### Opening a pull request
+
+We recommend using [Git](https://git-scm.com/) to commit changes. If you prefer GUIs, you can use [GitHub Desktop](https://desktop.github.com/) instead. You can also use the GitHub website if that's easier, but the other options speed up some of the process.
```

Comment:
Should we have the GitHub website listed as an option?

Reply:
I think it's ok as is

Reply:
```suggestion
We recommend using [Git](https://git-scm.com/) to commit changes. If you prefer GUIs, you can use [GitHub Desktop](https://desktop.github.com/) instead. If you don't want to install anything, you can use the GitHub website, but we recommend one of the other options if possible.
```